bin/_nemo

```diff
@@ -15,7 +15,7 @@ var log;
 var error;
 var server;
 var scaffold;
-var prefontaine;
+var engine;
```

Comment:
👍 love it. In hindsight I can see now how the runner analogy can work, but more boring names like this seem better for grok-ing the code quickly and making contributions.